python/python/glide/async_commands/redis_modules/json/commands.py

```diff
@@ -0,0 +1,109 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+import json
+from typing import Dict, List, Optional, Union, cast
+
+from glide.async_commands.core import ConditionalChange
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+TJson = Union[str, int, float, bool, None, List["TJson"], Dict[str, "TJson"]]
+
+
+class Json:
```

Comment:
This class doesn't save any state, not sure what's the gain of using it as a class rather than just a [module](https://docs.python.org/3/tutorial/modules.html), e.g.:
### json.py
```
def set():
    ...

def get():
    ...
```
### main.py
```
import json

json.set()
json.get()